samples/go/demo-server/web_server.go

```diff
@@ -152,3 +155,10 @@ func notFound(w http.ResponseWriter, r *http.Request) {
 	newParams = append(newParams, params...)
 	hndl(w, r, newParams)
 }
+
+func handleBaseGet(w http.ResponseWriter, req *http.Request, params httprouter.Params) {
+	d.PanicIfTrue(req.Method != "GET", "Expected get method.")
+
+	fmt.Fprintf(w, nomsBaseHtml)
```

Comment:
It does buffer them, but I think you're right that, in general, it's better in the other direction. I'll change it here and in the original code submitted by @jangie.